spec/services/api/create_params_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe Api::CreateParams do
+  describe "#permissions" do
+    it "allows scalar model attributes" do
+      pl_permissions = Api::CreateParams.new(Api::Model.new('plant_line'), {}).permissions
+      # FIXME plant_variety_name - shouldn't it be removed?
```

Comment:
Unfortunately you're correct - here's your problem:
<pre>
bip_development=# SELECT COUNT(*) FROM plant_lines WHERE plant_variety_name NOT IN (SELECT plant_variety_name FROM plant_varieties);
count 
-------
 15626
(1 row)
</pre>
It is safe to omit this column in all new `PlantLine` records (and thus not include it in any input forms) but existing data should retain it.